tests/pathhelp.pm

```diff
@@ -372,7 +372,15 @@ sub sys_native_abs_path {
         # Path is in relative form. Resolve relative directories in Unix form
         # *BEFORE* converting to Win32 form otherwise paths like
         # '../../../cygdrive/c/windows' will not be resolved.
-        my $cur_dir = `pwd -L`;
+
+        my $cur_dir;
+        # MSys shell has built-in command.
+        if($^O eq 'msys') {
+            $cur_dir = `bash -c 'pwd -L'`;
```

Comment:
why this

Reply:
See commit description.

Reply:
ah I see that's perl specific. I can do `pwd -L` by itself from the shell but it doesn't work from perl. what about $ENV{PWD}

Reply:
That is not Perl specific, msys1 pwd binary just does not support -L so I use the one built into the shell. The environment variable does not return the path in the correct msys specific format/style and is dependent on the Perl interpreter.